src/core/dev/cq_mgr.cpp

```diff
@@ -42,7 +42,7 @@
 #include <util/valgrind.h>
 #include "util/instrumentation.h"
 #include <sock/sock-redirect.h>
-#include "sock/sock-app.h"
+#include <sock/sock-app.h>
```

Comment:
Is this change required? I'd prefer to `"header.h"` for XLIO headers and `<header.h>` for the systems'.